doc/build-osx.md

```diff
@@ -20,23 +20,23 @@ Dependencies
 
 See [dependencies.md](dependencies.md) for a complete overview.
 
-If you want to build the disk image with `make deploy` (.dmg / optional), you need RSVG
+If you want to build the disk image with `make deploy` (.dmg / optional), you need RSVG.
```

Comment:
`If you want to build the disk image with `make deploy` (.dmg / optional), you need RSVG:`